sync/task_pool.go

```diff
@@ -84,14 +84,21 @@ func WithTaskPoolTaskQueueNumber(number int) TaskPoolOption {
 /////////////////////////////////////////
 
 // task t
-type task func()
+type runnable func() (interface{}, error)
```

Comment:
the runnable is not compatible with previous task definition

Reply:
yes , i see, once update. i will fix it  in dubbogo

Reply:
fix it